### ISSUE TITLE
readthedocs build error

### ISSUE THREAD

**Issue by pnorbert** [ornladios/ADIOS2#2243]
The documentation  build has been failing for a while now. Can someone please fix that?

[https://readthedocs.org/projects/adios2/builds/](https://readthedocs.org/projects/adios2/builds/)

```
ImportError: cannot import name 'math' from 'sphinx.addnodes' (/home/docs/checkouts/readthedocs.org/user_builds/adios2/conda/latest/lib/python3.7/site-packages/sphinx/addnodes.py)

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/adios2/conda/latest/lib/python3.7/site-packages/sphinx/ext/mathbase.py", line 16, in <module>
    from sphinx.addnodes import math, math_block as displaymath  # NOQA  # to keep compatibility
ImportError: cannot import name 'math' from 'sphinx.addnodes' (/home/docs/checkouts/readthedocs.org/user_builds/adios2/conda/latest/lib/python3.7/site-packages/sphinx/addnodes.py)
The full traceback has been saved in /tmp/sphinx-err-u0l9v897.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

**Comment by williamfgc** [comment]
@pnorbert let me take a look, I had those sphinx broken API issues in the past.

**Comment by williamfgc** [comment]
No luck with #2244 , seems other issues appear with breathe. I might just get rid of the breathe package or look for a replacement. It is too unstable.

**Comment by chuckatkins** [comment]
Seems to be fixed now with #2248.  PR building is also enabled wso that should keep it up to date and working.
